src/attr/_make.py

```diff
@@ -1068,6 +1068,32 @@ def fields(cls):
     return attrs
 
 
+def fields_dict(cls):
+    """
+    Returns an OrderedDict of ``attrs`` attributes for a class, whose keys are
```

Comment:
Ugh I’ve just noticed the “Returns” in `fields`! Please make both a “Return”, thanks!